lib/plausible_web/controllers/site_controller.ex

```diff
@@ -708,6 +708,23 @@ defmodule PlausibleWeb.SiteController do
     |> redirect(external: Routes.site_path(conn, :settings_integrations, site.domain))
   end
 
+  def export(conn, _params) do
+    %{site: site, current_user: user} = conn.assigns
+
+    Oban.insert!(
+      Plausible.Workers.ExportCSV.new(%{
+        "site_id" => site.id,
+        "email_to" => user.email,
+        "s3_bucket" => Plausible.S3.exports_bucket(),
+        "s3_path" => "Plausible-#{site.id}.zip"
+      })
+    )
+
+    conn
+    |> put_flash(:info, "SCHEDULED. WAIT FOR MAIL")
```

Comment:
```suggestion
    |> put_flash(:success, "SCHEDULED. WAIT FOR MAIL")
```

as we lack flash rendering for `:info`.